model/autosklearn/dffml_model_autosklearn/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+**Ubuntu Installation**
+
+To provide a C++11 building environment and the lateste SWIG version on Ubuntu, run:
+
+.. code-block:: console
+    $ sudo apt-get install build-essential swig
+
+Install other PyPi dependencies with
```

Comment:
You shouldn't need this section on the `requirements.txt`, those should be installed when the pypi package of `auto-sklearn` is installed

Reply:
yes, ideally that should happen,
but what happens is it would throw a lot of error on the screen and many wheels installation fails, pip again tries to build it from scratch then it gets installed. (installation does takes place but it throws a lot of errors and takes time)

installing it's dependencies first and then installing the auto-sklearn looks clean and works fast too.

So i thought maybe mention otherwise it works fine both ways.

Reply:
Okay great. That's odd, good job finding this workaround that makes it clean!